packages/api/amplify_api/example/integration_test/util.dart

```diff
@@ -138,6 +138,23 @@ Future<Blog> addBlog(String name) async {
   return blog;
 }
 
+// declare utility which creates post with title and blog as parameter
+Future<Post> addPost(String name, Blog blog) async {
```

Comment:
Nice util! Request: Could you please remove similar code from end of `addPostAndBlog` and just call this method from there to DRY?